src/TritonRoute/src/global.h

```diff
@@ -65,6 +65,7 @@ extern bool ENABLE_VIA_GEN;
 // extern int TEST;
 extern fr::frLayerNum VIAINPIN_BOTTOMLAYERNUM;
 extern fr::frLayerNum VIAINPIN_TOPLAYERNUM;
+extern fr::frLayerNum VIAINPIN_DEFAULT_BOTTOMLAYERNUM;
```

Comment:
This is really just a constant, there is no need to make it a global variable.

Reply:
But I use this in FlexPA::prepPoint_pin_checkPoint_via

Reply:
That is a question of scope; it is a constant not a variable.  Just put in the header:

constexpr fr::frLayerNum VIAINPIN_DEFAULT_BOTTOMLAYERNUM = std::numeric_limits<frLayerNum>::max();